package.json

```diff
@@ -54,7 +54,8 @@
     "just-test": "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
     "test": "npm run build-test && npm run just-test",
     "cover": "npm run build-test && nyc npm run just-test",
-    "prepublish": "npm run transpile"
+    "prepublish": "npm run transpile",
+    "flow": "flow check"
```

Comment:
I would rather not add this script because we don't have it in other plugins, and if we want it we should change all plugins at once.

Reply:
Sure, I'll remove for conformity sake